style.css

```diff
@@ -97,8 +104,175 @@ body {
   display: flex;
   margin-left: 24px;
   margin-top: 12px;
+  list-style-type: none;
 }
 
 .sc-item {
   margin-left: 20px;
 }
+
+.works {
+  margin-top: 136px;
+  margin-bottom: 114px;
+}
+
```

Comment:
- [ ] **[OPTIONAL] Good job on replicating the Figma design :clap: 👏** 
   However, It would be better to replicate the body background-color that of the Figma design, this will give a clear view and a user-friendly design 👍 👍 

**Your design currently** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/219605420-a8a98a3c-981d-4746-bcf1-c3f0b0d17f2f.png) | ![image](https://user-images.githubusercontent.com/102757126/219429819-a1fc0989-dd85-4a50-ab51-34ac1f06a355.png)

- [ ] **[Optional]** You may also consider replicating the space between the **header section** and **Work section** to replicate as much as possible the Figma design 👍 👍